### ISSUE TITLE
New Interface Design

### ISSUE THREAD

**Issue by ArkDexOficial** [ChronosWS/asma#64]
Creates an interface similar to the old ASM.

**Comment by ArkDexOficial** [comment]
This was the most similar asthma I have met so far with the old one.

Video example, stopped working for over 2 months without changing anything.
If you find someone who makes an interface to interconnect with what has already been done, it will be impeccable.


[screen-capture (7).webm](https://github.com/ChronosWS/asma/assets/156136172/fec31001-e3e0-46d8-aa2f-2f4cb42eddb5)

**Comment by tuvok81** [comment]
> Creates an interface similar to the old ASM.

yes this was wounderfull .. the same ASMA Server Manager with the Old Design was nice ..

**Comment by wisemonk** [comment]
This something you are still working on ? as I could have ago at the interface
